Content.Server/GameObjects/Components/Chemistry/InjectorComponent.cs

```diff
@@ -106,7 +107,7 @@ private void Toggle(IEntity user)
         /// Called when clicking on entities while holding in active hand
         /// </summary>
         /// <param name="eventArgs"></param>
-        void IAfterInteract.AfterInteract(AfterInteractEventArgs eventArgs)
+        public async Task AfterInteract(AfterInteractEventArgs eventArgs)
         {
```

Comment:
Kinda weird to change these to public

Reply:
I re-compiled the code several times and fixed the issues which croped up with certain methods being public, this one wasn't one of them.
Maybe the right thing to do is to make it just async but it didn't cause any issues